### PR TITLE
added spring-amqp pages and links from menu

### PR DESCRIPTION
**Pull request by wxlund**
added in spring-amqp tutorials to the list of options for rabbitmq clients.

### CODE REVIEW

**Comment by michaelklishin** [comment]
I'm guessing this was meant to be submitted against the `live` branch. Since it's a single commit I will cherry-pick. Thank you!

File: site/tutorials/tutorial-four-spring-amqp.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+We'll use this model for our logging system. Instead of `fanout` we'll
+send messages to a `direct` exchange. We will supply the log severity as
+a `routing key`. That way the receiving program will be able to select
+the severity it wants to receive. Let's focus on emitting logs
+first.
+
+As always, we need to create an exchange first:
+
+    :::java
```
Comment (michaelklishin) [inline site/tutorials/tutorial-four-spring-amqp.md]: This is not how code examples are formatted in tutorial 1 or Java tutorial 3.

File: site/tutorials/tutorial-four-spring-amqp.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    
+            channel.close();
+            connection.close();
+        }
+        //..
+    }
+
+The code for `ReceiveLogsDirect.java`:
+
+    #!java
```
Comment (michaelklishin) [inline site/tutorials/tutorial-four-spring-amqp.md]: This does not use any code formatting, please use Java tutorials as examples.

**Comment by michaelklishin** [comment]
Sorry but it's quite far from being ready to go live.

I had to create [a new branch](https://github.com/rabbitmq/rabbitmq-website/tree/spring-amqp-tutorials-live) off of `live` and cherry-pick both this PR's commit and https://github.com/rabbitmq/rabbitmq-website/commit/1c28a8bdb1cf5da66edd8ca373ef4535a65ebeb9 there.

This PR doesn't use the same code highlighting mechanism as our tutorials in `live`. Please fix that on the new branch.

**Comment by wxlund** [comment]
I somehow checked in the wrong work.  It wasn't close to what I'd finished.  Really sorry about that and am not sure what I did as the current branch you created isn't close to my work.  Should I try repushing or copying my work over to this new branch you created?

**Comment by michaelklishin** [comment]
Just use the branch I pushed. It seems to have all the tutorials in place, just uses outdated code highlighting.

**Pull request closed without merging**